### PR TITLE
Product condition choice

### PR DESCRIPTION
**Pull request by Traumflug**
This is @Nobodaddy's pull requests put together in order to make them pass Travis.

Original PRs: #260, #261 

This change makes sense together with this theme change: https://github.com/thirtybees/community-theme-default/pull/6

### CODE REVIEW

**Comment by Nobodaddy** [comment]
Many thanks. :)

File: controllers/admin/AdminPPreferencesController.php [hunk 1]
```diff
@@ -225,6 +225,14 @@ public function __construct()
                         'required'   => false,
                         'type'       => 'bool',
                     ],
+                    'PS_SHOW_CONDITION'             => [
+                        'title'      => $this->l('Show condition'),
+                        'hint'       => $this->l('Show/Hide conditions dropdown list on product page.'),
```
Comment (Traumflug) [inline controllers/admin/AdminPPreferencesController.php]: This hint talks about a "dropdown list", but AFAIK product condition is plain text on the product page.

Comment (Nobodaddy) [inline controllers/admin/AdminPPreferencesController.php]: Sorry, my bad. As I don't have no access here please change this hint to
*Show/Hide condition  on product page.*

**Comment by Dh42** [comment]
The other pull makes more sense now, I thought it was missing the logic to actually work.

File: controllers/admin/AdminPPreferencesController.php (hunk 1 above)
Comment (Traumflug) [inline controllers/admin/AdminPPreferencesController.php]: Done by adjusting the original commit. Thanks.

**Comment by Traumflug** [comment]
So you agree with this merging this request, @Dh42 ?

**Comment by Dh42** [comment]
Sure, I don't have any issues with it.

**Comment by Traumflug** [comment]
Excellent. Thank you, everybody!

**Comment by Traumflug** [comment]
Github wrote:
> merged commit 84f319e into thirtybees:1.0.x

Just for the record, no merge happened. *1.0.x* was rebased to that commit to ensure a straight history *without* merge points. This eases bisecting and working with topic branches.

**Pull request merged**